llama_stack/cli/stack/build.py

```diff
@@ -137,10 +137,16 @@ def _run_stack_build_command_from_build_config(
             if build_config.image_type == "conda"
             else (f"llamastack-{build_config.name}")
         )
-        cprint(
-            f"You can now run `llama stack configure {configure_name}`",
-            color="green",
-        )
+        if build_config.image_type == "conda":
+            cprint(
+                f"You can now run `llama stack configure {configure_name}`",
+                color="green",
+            )
+        else:
+            cprint(
+                f"You can now run `llama stack run {build_config.name}` or `llama stack configure {configure_name} to re-configure docker run config.`",
```

Comment:
After building a docker image with built-in configure. We could directly `llama stack run` w/o needing to run `configure` again.

Reply:
The wording could make it a bit more clear that if you run `llama stack configure`, you need to run `llama stack run` to get the right config loaded. Otherwise, they need to know how to mount the config into the container properly.

Reply:
What do you mean by "if you run llama stack configure, you need to run llama stack run to get the right config loaded"? 

There's 2 dev flow for docker: 

1. llama stack build --> llama stack run (to use the configured run file generated during build)
2. llama stack build --> llama stack configure (to reconfigure) --> llama stack run. 

(1) is from this PR. (2) is what you see with having to configure twice.

Reply:
Yes - understood and agreed on the 2 flows.

The part that I thought was slightly confusing is:

> to re-configure docuer run config

which implies a bit that you can use `docker run` after `configure`, but it's a bit more complicated than that since you need to know the right set of options to load that new config in your container.

Here's what I propose:

```suggestion
                f"You can now run `llama stack run {build_config.name}` or `llama stack configure {configure_name}`.",
```

Maybe we should write some new docs on using containers that covers all of the details of the options.

Reply:
Got you, updated w/ new message.